xml/installation-installation-gui_installer.xml

```diff
@@ -238,26 +237,147 @@ controller1,192.168.110.3,b2:72:8d:ac:7c:6f,RACK1,HP-DL360-4PORT,CONTROLLER-ROLE
+    </para>
+<screen>
+10.10.10.10 &susemgr;.test.domain
+</screen>
+   </step>
+  </procedure>
+  <para>
+   Use the following steps to import an &oneview; certificate.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Retrieve the <literal>sessionID</literal>.
+    </para>
+<screen>
+curl -k -H "X-Api-Version:500" -H "Content-Type: application/json" \
+-d '{"userName":<replaceable>ONEVIEW USERNAME</replaceable>, "password":<replaceable>ONEVIEWPASSWORD</replaceable>, \
+"loginMsgAck":"true"}' https://<replaceable>ONEVIEW_MANAGER_URL:PORT</replaceable>/rest/login-sessions
+</screen>
+    <para>
+     The response will be similar to:
+    </para>
+<screen>
+{"partnerData":{},"sessionID":"LTYxNjA1O1NjkxMHcI1b2ypaGPscErUOHrl7At3-odHPmR"}
+</screen>
+   </step>
+   <step>
+    <para>
+     Retrieve a Certificate Signing Request (CSR) using the
+     <replaceable>sessionID</replaceable> from the previous step.
+    </para>
+<screen>
+<?dbsuse-fo font-size="0.7em"?>	
+curl -k -i -H "X-Api-Version:500" -H <replaceable>sessionID</replaceable> <replaceable>ONEVIEW_MANAGER_URL</replaceable>/rest/certificates/ca &gt; <replaceable>CA_NAME.csr</replaceable>
+</screen>
+   </step>
+   <step>
+    <para>
+     Follow instructions in your &oneview; User Guide to validate the CSR and
+     obtain a signed certificate
+     (<filename><replaceable>CA_NAME</replaceable>.crt</filename>).
+    </para>
+   </step>
+   <step>
+    <para>
+     Copy the <filename>.crt</filename> file to the proper location on the
+     &lcm;.
+    </para>
+<screen>
+cd /etc/pki/trust/anchors
+/etc/pki/trust/anchors > sudo cp ~/data/<replaceable>CA_NAME</replaceable>.crt .
+</screen>
+   </step>
+   <step>
+    <para>
+     Install the certificate.
+    </para>
+<screen>
+sudo update-ca-certificates
+</screen>
+   </step>
+   <step>
+    <para>
+     Follow instructions in your &oneview; User Guide to import the
+     <replaceable>CA_NAME</replaceable>.crt certificate into &oneview;.
+    </para>
+   </step>
+   <step>
+    <para>
+     Add <replaceable>&oneview; host IP address</replaceable> if
+     <replaceable>&oneview;.test.domain</replaceable> is not reachable by DNS.
+    </para>
+<screen>
+sudo vi /etc/hosts
+</screen>
+    <para>
+     Add <replaceable>&oneview; host IP address</replaceable>
+     <replaceable>&oneview;.test.domain</replaceable> For example:
+    </para>
+<screen>
+10.84.84.84  &oneview;.test.domain 
+</screen>
+   </step>
+  </procedure>
  </section>
--->
  <section>
   <title>Running the &install_ui;</title>
   <para>
    Deploying the cloud to your servers will reconfigure networking and firewall
    rules on your cloud servers. To avoid problems with these networking changes
    when using the &install_ui;, we recommend you run a browser directly on your
-   &lcm; node and point it to <uri>http://localhost:3000</uri>.
+   &lcm; node and point it to
+   <uri>http://localhost:3000</uri>
+   .
```

Comment:
Don't put that dot on an extra line, that breaks formatting (that is an issue with daps-xmlformat -- I just fixed that in https://github.com/openSUSE/daps/commit/ac18bdbdafb4a0f4722baa933c8ac40d05e6e205 but it might be some time before that fix is in a package, until then, apply the same fix to `/etc/daps/docbook-xmlformat.conf`).